### PR TITLE
Updated the Page container

### PR DESCRIPTION
**Pull request by stealthanthrax**
## Changelog
Essentially the same or as https://github.com/MLH-Fellowship/react-native/pull/103

Just pointing to a different branch.

## Summary
I have created a basic use case container as mentioned in #102  , which will serve as a template for all the use case screens.

## Changelog

A demo screen would look like this at the current moment.
Screenshot 2020-07-01 at 4 28 21 AM

## Test Plan
I built the app on XCode and got the usual warnings in the bundler.

```
To reload the app press "r"
To open developer menu press "d"

[Wed Jul 01 2020 04:37:21.504]  BUNDLE  RNTester/js/RNTesterApp.ios.js 

[Wed Jul 01 2020 04:37:23.881]  ERROR    Warning: DatePickerIOS has been merged with DatePickerAndroid and will be removed in a future release. It can now be installed and imported from '@react-native-community/datetimepicker' instead of 'react-native'. See https://github.com/react-native-community/datetimepicker
[Wed Jul 01 2020 04:37:23.882]  ERROR    Warning: SegmentedControlIOS has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/segmented-control' instead of 'react-native'. See https://github.com/react-native-community/segmented-control
[Wed Jul 01 2020 04:37:23.883]  ERROR    Warning: MaskedViewIOS has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/masked-view' instead of 'react-native'. See https://github.com/react-native-community/react-native-masked-view
[Wed Jul 01 2020 04:37:23.884]  ERROR    Warning: Picker has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/picker' instead of 'react-native'. See https://github.com/react-native-community/react-native-picker
[Wed Jul 01 2020 04:37:23.886]  ERROR    Warning: PickerIOS has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/picker' instead of 'react-native'. See https://github.com/react-native-community/react-native-picker
[Wed Jul 01 2020 04:37:23.886]  ERROR    Warning: ProgressViewIOS has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/progress-view' instead of 'react-native'. See https://github.com/react-native-community/progress-view
[Wed Jul 01 2020 04:37:23.887]  ERROR    Warning: Slider has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/slider' instead of 'react-native'. See https://github.com/react-native-community/react-native-slider
[Wed Jul 01 2020 04:37:23.888]  ERROR    Warning: AsyncStorage has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/async-storage' instead of 'react-native'. See https://github.com/react-native-community/async-storage
[Wed Jul 01 2020 04:37:23.888]  ERROR    Warning: Clipboard has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/clipboard' instead of 'react-native'. See https://github.com/react-native-community/clipboard
[Wed Jul 01 2020 04:37:23.889]  ERROR    Warning: PushNotificationIOS has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/push-notification-ios' instead of 'react-native'. See https://github.com/react-native-community/push-notification-ios
[Wed Jul 01 2020 04:37:23.890]  ERROR    Warning: ProgressBarAndroid has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/progress-bar-android' instead of 'react-native'. See https://github.com/react-native-community/progress-bar-android
[Wed Jul 01 2020 04:37:23.891]  LOG      Running "RNTesterApp" with {"rootTag":1,"initialProps":{}}
```

### CODE REVIEW

**Comment by stealthanthrax** [comment]
Hey @ani4aniket . I was totally confused after the last meeting. So, I used [Cunningham's Law](https://meta.wikimedia.org/wiki/Cunningham%27s_Law) , will do it by tonight.

**Comment by chirag-singhal** [comment]
@stealthanthrax Main title and note are not aligned to center. Also can you add some padding between the title, note and rendered example.

**Comment by chirag-singhal** [comment]
@stealthanthrax Also the example page title like 'Basic Gesture Handler' is not showing up.

File: packages/RNTester/js/components/ExamplePage.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      source={imagePaths.android}
+    />
+  ) : null;
+
+  const appleImage = props.ios ? (
+    <Image style={{height: 35, width: 30, margin: 2}} source={imagePaths.ios} />
+  ) : null;
+
+  const docsImage = <Image source={imagePaths.docs} />;
+  // const children = React.Children.map(props.children, (child, index) => (
```
Comment (anku255) [inline packages/RNTester/js/components/ExamplePage.js:54]: Do these comments serve any purpose?

**Comment by stealthanthrax** [comment]
@anku255 @chirag-singhal , I have fixed the issues, can you have a look again?

**Comment by anku255** [comment]
@stealthanthrax There is some scrolling issue with the new Page Container.

For example, this is the farthest I can scrolling the Alert page, but there is one more example that is not showing up. 


Screenshot 2020-07-21 at 5 50 36 PM

This is the problem in every screen. Even in SwitchExample, you can see that a portion of the example is getting cut off.

Screenshot 2020-07-21 at 5 52 27 PM

**Comment by stealthanthrax** [comment]
@anku255 , by cutting off , do you mean that it is not fitting in the screen or is the vertical scrolling not going beyond a certain point?

**Comment by anku255** [comment]
> @anku255 , by cutting off , do you mean that it is not fitting in the screen or is the vertical scrolling not going beyond a certain point?

Yeah, it stops scrolling after a certain point (even there is content below it).

**Comment by anku255** [comment]
@stealthanthrax If you don't mind I would take a look at this. We need to get this merged first to unblock #164.

**Comment by anku255** [comment]
@stealthanthrax I have pushed a change. It fixes the issue for me. Can you please take a look at this?

**Comment by chirag-singhal** [comment]
@anku255 there is a gap at the bottom of each example page, is that intentional for navigation bar?

**Comment by stealthanthrax** [comment]
@anku255 , leaving space for the navbar is not the right way to approach this . As navbar is supposed to come over the top of the contents for aesthetics and matching with the design files.

**Comment by stealthanthrax** [comment]
Just fixing the padding should fix this. I'll make a commit ASAP

**Comment by stealthanthrax** [comment]
@anku255 , have a look again. If it works well, we'll merge it.

**Comment by anku255** [comment]
@stealthanthrax Doesn't seem to be working for me. There should be one more example in this screen. :(

**Comment by manyaagarwal** [comment]
Also, we need to add the search bar to the header before merging this. Currently the search bar scrolls away with the list of examples.

**Comment by stealthanthrax** [comment]
@manyaagarwal , that's not in the scope of this PR. We'll handle the search bar in a different one.

**Comment by stealthanthrax** [comment]
As search bar requires a lot of styling anyway.

**Comment by manyaagarwal** [comment]
@stealthanthrax We should atleast be placing the search bar in the header without the new styling.

**Comment by stealthanthrax** [comment]
@manyaagarwal , this PR just the implementation of a *container* not for the UI components.

**Comment by manyaagarwal** [comment]
@stealthanthrax I get that, but the search bar should not be a part of the examples list and should be fixed. I think that counts as a container requirement. But we can take it to a separate PR if it's going to take a lot of time.

**Comment by stealthanthrax** [comment]
Yes, I would suggest we take that in a different PR.

**Comment by stealthanthrax** [comment]
Merging it now 😄

**Pull request merged**